### PR TITLE
Testing with larger data sets and pagination

### PR DESCRIPTION
**Pull request by annegentle**
Also requests 2.28.0

### CODE REVIEW

File: gh_search.py [hunk 1]
```diff
@@ -17,58 +17,52 @@ def search_github(keyword, filetype):
...
         if filetype == 'any':
             query = f'"{keyword}" org:{config.gh_orgname}'
-            result = g.search_code(query, order='desc')
-
-            max_size = 100
-            
-            print(f'Found {result.totalCount} file(s) with {keyword}')
+            searchresults = g.search_code(query, order='desc')
+            # search_code returns a paginated_list https://pygithub.readthedocs.io/en/latest/utilities.html?highlight=pagination#pagination
+            #print(f'Found {searchresults.totalCount} entries with {keyword}')
```
Comment (juliaferraioli) [inline gh_search.py:25]: Question: should lines 25/26 be log statements rather than comments?

**Review by juliaferraioli — CHANGES_REQUESTED** [review]
Tested and seems to work for me. Only feedback is that the debugging print statements should be either removed or turned into logging statements prior to merging.

File: gh_search.py (hunk 1 above)
Comment (annegentle) [reply gh_search.py]: Agreed, updated with logging.

**Comment by annegentle** [comment]
Also tested with updated certifi in requirements.txt.

**Review by juliaferraioli — APPROVED** [review]
LGTM!

**Pull request merged**